apps/website/next.config.ts

```diff
@@ -30,6 +30,13 @@ const config: NextConfig = {
     dirs: ["."],
   },
   images: {
+    // limits the quality levels to limit cache variations for image optimization:
+    qualities: [50, 75, 100],
```

Comment:
Can we add 90 here as we use it in a few places? (100 is pass-through and doesn't get optimized at all)